app/core/config.py

```diff
@@ -61,3 +61,4 @@
 NF_PIPELINE_URL = config("NF_PIPELINE_URL")
 NF_WORK_DIR = config("NF_WORK_DIR")
 SEQERA_WORKFLOW_LAUNCH_URL = config("SEQERA_WORKFLOW_LAUNCH_URL")
+VEP_CONFIG_INI_PATH = config("VEP_CONFIG_INI_PATH")
```

Comment:
This should be exposed to all the environment via tools-api.secret  in newwebsite-config repo. It doesn't have side affect so I will add it.

Reply:
This is done